app/posts/claim-general-mentor-training-funding/2022-06-26-general-mentor-training-grant-guidance-and-claim-form.md

```diff
@@ -0,0 +1,123 @@
+
+- mentors to train take part in 20 hours of initial training and 6 hours of top-up training in subsequent years
+- ITT providers to have lead mentors or a lead mentorship team
+- trainees to take part in ’Intensive training and practice’ sessions (4 weeks for postgraduate courses and 6 weeks for undergraduate courses)
+
+## ITT reform funding
+
+To support providers and schools in meeting the new requirements the DfE is making grant funding available.
+
+For schools the funding is to help cover the cost of classroom cover while mentors train. In the 2024/25 academic year, general mentors will need to complete 20&nbsp;hours of initial mentor training. In subsequent years general mentors who have completed the 20&nbsp;hours will need 6&nbsp;hours of training.
+
+[See full details of the ITT reform funding.](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1077177/Initial_Teacher_Training__ITT__Reform_Funding_Guidance_160522.pdf)
+
+## Prototype
+
+One of the streams of work for the mentoring team is looking into how the grants will be distributed.
+
+We plan to distribute the funding for ITT providers through DfE’s normal business processes.
+
+Because the general mentor training grant is available to more than 20,000&nbsp;schools and we need specific information for each claim, a digital form is necessary to collect the data.
+
+To test our questions and assumptions about how the service will work, we built a guidance page and a form (in Google forms).
+
+### Guidance
+
+![GOV.UK page with summary of ITT general mentor training grant](general-mentor-training-grant-guidance.png "Apply for ITT general mentor training funding")
+
+### Claim form
+
+![Google form asking user for their email address](google-form/01-email-address.png "Claim form — email address")
+
+![Google form asking user for the URN of their school](google-form/02-school.png "Claim form — school unique reference number (URN)")
+
+![Google form asking user for number of mentors, hours of classroom cover needed, and hours of classroom cover paid for by the provider](google-form/03-mentors.png "Claim form — number of mentors and training time")
+
+![Google form asking user if each mentor trained at least one trainee](google-form/04-declaration.png "Claim form — each mentor trained at least one trainee")
+
+![Google form confirmation the form has been completed](google-form/05-confirmation.png "Claim form — confirmation")
+
+### Information required for a claim
+
+#### School details
+
+So we know who is making the claim and who to pay.
+
+The amount of funding available is adjusted for location. To pay the correct amount DfE will need to know where the school is.
+
+| **Location**    | **Amount** |
+|:---------------:|:----------:|
+| Inner London    |      £1072 |
+| Outer London    |       £965 |
+| Fringe          |       £902 |
+| Rest of England |       £876 |
```

Comment:
Both these columns are centred. Ideally they would be left and right aligned respectively.

Reply:
Yep — just learnt how to do that in markdown.